osu.Server.Spectator/Hubs/StatefulUserHub.cs

```diff
@@ -22,9 +22,12 @@ public abstract class StatefulUserHub<TClient, TUserState> : Hub<TClient>
     {
         protected readonly EntityStore<TUserState> UserStates;
 
+        private readonly Logger logger;
+
         protected StatefulUserHub(IDistributedCache cache, EntityStore<TUserState> userStates)
         {
-            this.UserStates = userStates;
+            logger = Logger.GetLogger(GetType().Name.Replace("Hub", string.Empty));
```

Comment:
While this creates a logger, `Logger.Storage` isn't set, meaning there won't be any log files created for this.